packages/tc-api-rest-handlers/lib/events/index.js

```diff
@@ -66,11 +87,16 @@ const logChanges = (
 		throw new Error(message);
 	}
 
-	const events = makeEvents(action, entity, relationships);
-	const publisher = createPublisher(adaptor);
-	publisher.publish(...events);
+	makeEvents(action, entity, relationships).forEach(event =>
+		emitter.emit(event.action, {
```

Comment:
Do we need one more property like `event.kind` in order to be easy to distinguish for subscribers which event should handle or not? Eventually, some other event-driven module will use this.

Reply:
what value would `event.kind` contain? `action` is already a verb, which suits being a name for an event